keyboards/oktiboard/readme.md

```diff
@@ -0,0 +1,12 @@
+# oktiboard
+
+* Keyboard Maintainer: QMK Community
+* Hardware Supported: Pro Micro
+    * 16 NeoPixel Led
```

Comment:
```suggestion
* Hardware Supported: Pro Micro, 16 NeoPixel LED
```

Reply:
Also adding the availability line would be nice, as at the moment its unclear if this is a handwire.

Reply:
I moved to under handwired folder and updated the readme file. Handwired information added.